src/core/entities/MRLightEntity.js

```diff
@@ -22,7 +22,7 @@ export class MRLightEntity extends MREntity {
      * @function
      * @description Callback function of MREntity - handles setting up this Light once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?